beetmoverscript/script.py

```diff
@@ -247,10 +249,32 @@ def list_bucket_objects(context, s3_resource, prefix):
 # move_beets {{{1
 async def move_beets(context, artifacts_to_beetmove, manifest):
     beets = []
+
     for locale in artifacts_to_beetmove:
+
+        installer_path = ''
+
+        # get path of installer beet
+        for artifact in artifacts_to_beetmove[locale]:
+            # import pdb; pdb.set_trace()
```

Comment:
Remove this line. Debugging lines, commented or not are not supposed to get into production.